api-consumption/src/main/java/org/zalando/nakadi/service/subscription/state/PartitionData.java

```diff
@@ -186,7 +186,7 @@ private CommitResult(final boolean seekOnKafka, final long committedCount) {
     CommitResult onCommitOffset(final NakadiCursor offset) {
         boolean seekKafka = false;
         if (comparator.compare(offset, sentOffset) > 0) {
-            LOG.error("Commit in future: current: {}, committed {} will skip sending obsolete data", sentOffset,
+            LOG.error("Commit in future: new: {}, sent: {}, committed: {}. Will skip sending obsolete data", offset, sentOffset,
```

Comment:
what does `new` means here ?

Reply:
It's the offset which is currently being tried to commit. I felt that would be clearer than "current", but feel free to suggest a better wording.

Reply:
I've applied @antban's suggestion ("received").